mtools/mloginfo/sections/connection_section.py

```diff
@@ -37,10 +39,28 @@ def run(self):
 
         ip_opened = defaultdict(lambda: 0)
         ip_closed = defaultdict(lambda: 0)
+        
         socket_exceptions = 0
 
+        genstats = self.mloginfo.args['stats']
+        if genstats:
+            genstats = True
+            connections_start = defaultdict(lambda: 0)
+            ipwise_sum_durations = defaultdict(lambda:0)
+            ipwise_count = defaultdict(lambda:0)
+            ipwise_min_connection_duration = defaultdict(lambda:9999999999)
+            ipwise_max_connection_duration = defaultdict(lambda:-1)
+
+            min_connection_duration = 9999999999
```

Comment:
You could declare `min_connection_duration` and `max_connection_duration` before `ipwise_min` and `ipwise_max` declaration to avoid repeating the same values. i.e. `lambda: min_connection_duration`

[Optional] replace defaultdict with normal dict to be explicit in your dictionary key declaration.